src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/CreateResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,141 @@
+                        "title": "Data Validation Failed",
+                        "status": 400,
+                        "correlationId": null,
+                        "validationErrors": {
+                        "$.codeValue": [
+                                "CodeValue is required and should not be left empty."
+                            ]
+                        }
+                    }
+                  """
+        @ignore
+        Scenario: Verify error handling with POST using empty body
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                    }
+                  """
+             Then it should respond with 403
+              And the response body is
+                  """
+                    {
+                        "detail": "Data validation failed. See 'validationErrors' for details.",
+                        "type": "urn:ed-fi:api:bad-request:data",
+                        "title": "Data Validation Failed",
+                        "status": 400,
+                        "correlationId": null,
+                        "validationErrors": {
+                            "$.codeValue": [
+                                "CodeValue is required."
+                                ],
+                            "$.namespace": [
+                                "Namespace is required."
+                                ],
+                            "$.shortDescription": [
+                                "ShortDescription is required."
+                                ]
+                        }
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify POST of existing record without changes
+                          #an existing record to
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                             "codeValue": "Sick Lave",
+                             "description": "Sick Leave",
+                             "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                             "shortDescription": "Sick Leave"
+                         }
+                  """
+             Then it should respond with 200
+              And the record can be retrieved with a GET request
+
+        @ignore
+        Scenario: Verify POST of existing record (change non-key field) works
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Lave",
+                        "description": "Sick Leave Edit",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 200
```

Comment:
I just thought of another scenario: add an "id" to the request body. Should get a 400 response with message like:

```json
{
  "detail": "The request data was constructed incorrectly.",
  "type": "urn:ed-fi:api:bad-request:data",
  "title": "Data Validation Failed",
  "status": 400,
  "correlationId": "cac9bef8-2f4e-47b2-bc31-0ac7e7db3d7f",
  "errors": [
    "Resource identifiers cannot be assigned by the client. The 'id' property should not be included in the request body."
  ]
}
```

Reply:
Yes, we already cover that scenario in the UpdateResource file, let me know if you want me to move it.

Reply:
We need to cover it in the CreateResourcesValidation file, because it is "against the rules" to send a POST request with  an `id` in it, whereas it is _required_ to include the `id` when sending a PUT request.